internal/observers/docker/labels.go

```diff
@@ -9,18 +9,23 @@ import (
 )
 
 var labelConfigRegexp = regexp.MustCompile(
-	`^agent.signalfx.com.` +
+	`^agent.signalfx.com\.` +
```

Comment:
is this a breaking change?  I noticed you had to update the test labels.

Reply:
It was always documented with the dots but was allowing anything through inadvertently.  So anybody using slashes was going against the docs.